src/index.js

```diff
@@ -50,6 +54,8 @@ export function parseValue(value) {
     return parseList(value);
   } else if (_.isPlainObject(value)) {
     return parseMap(value);
+  } else if (ESCAPE_STRINGS && _.isString(value)) {
```

Comment:
Great point! I think the option should be passed to `parseValue` to keep that function pure. 

```javascript
export function parseValue(value, { escapeStrings = false }) {
  ...
```